public/index.html

```diff
@@ -9,65 +9,7 @@
-            id="brandName" alt="Google Shopping">
-          <span id="shopping">Shopping</span>
-        </div>
-      </div>
-      <div id="searchSpace">
-        <div id="outline">
-          <div id="magGlass"></div>
-          <div id="searchText">Search Google Shopping</div>
-        </div>
-      </div>
-      <div id="optionsSpace">
-        <div id="cartSpace">
-          <span id="cart"></span>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div id="productInfoTop">
-    <img
-      src="https://mybundlefile.s3.us-east-2.amazonaws.com/Screen+Shot+2020-11-06+at+7.51.55+PM.png" id="topInfoStatic" alt="conectFour">
-  </div>
-  <div id="container">
-    <div id='productImagesService'></div>
-    <div id='product-information'></div>
-    <!-- <div id="app">
-      <ul>
-        <li>
-          <span>Classic Connect 4 game is disc dropping fun</span>
-        </li>
-        <li>
-          <span>Choose yellow or red discs. For 2 players</span>
-        </li>
-        <li>
-          <span>When you get 4 discs in a row you win</span>
-        </li>
-        <li>
-          <span>Includes grid, 2 legs, slider bar, 21 red discs, 21 yellow discs and instructions</span>
-        </li>
-      </ul>
-    </div> -->
-    <div id="price-card"></div>
-  </div>
-  <div id="productSpecification">
-    <img
-      src="https://mybundlefile.s3.us-east-2.amazonaws.com/Screen+Shot+2020-11-06+at+7.56.29+PM.png" id="specificationStatic" alt="conectFourSpec">
-  </div>
-  <div id="reviews-service"></div>
-  <script src="https://bundlefile.s3.us-east-2.amazonaws.com/bundle-product-information-service.js"></script>
-  <script src="http://18.220.123.159/bundle-seller-catalog.js"></script>
-  <script src="https://teamstructureshopping.s3.amazonaws.com/bundle/bundle-images-service.js"></script>
-  <!-- <script src="https://mybundlefile.s3.us-east-2.amazonaws.com/bundle-seller-catalog.js"></script> -->
-  <script src="https://rpt23-reviews-service.s3.us-east-2.amazonaws.com/bundle-reviews-service.js"></script>
+  <div id="price-card"></div>
```

Comment:
Notice that there is no `<script>` tag here.  That's because I manually copied the bundle file from the server to the public folder in this proxy.  See added file `public/bundle-seller-catalog.js` above.